monkey/infection_monkey/monkey.py

```diff
@@ -117,6 +122,20 @@ def _get_arguments(args):
 
         return opts
 
+    def _get_server(self):
+        servers_iterator = (s for s in self._opts.servers)
+        server = find_server(servers_iterator)
+        if server:
+            logger.debug(f"Default server set to: {server}")
```

Comment:
We're not really setting anything here, and that's not terribly useful to the user. 

```suggestion
            logger.info(f"Successfully connected to the island via {server}")
```